src/main/resources/META-INF/rewrite/jakarta-ee-10.yml

```diff
@@ -63,15 +64,51 @@ recipeList:
     - org.openrewrite.java.ChangeMethodName:
         methodPattern: jakarta.xml.soap.SOAPElementFactory create(String,..)
         newMethodName: createElement
-        ignoreDefinition: false
+        ignoreDefinition: true
     - org.openrewrite.java.ChangeMethodName:
         methodPattern: jakarta.xml.soap.SOAPElementFactory create(jakarta.xml.soap.Name)
         newMethodName: createElement
-        ignoreDefinition: false
+        ignoreDefinition: true
     - org.openrewrite.java.ChangeType:
         oldFullyQualifiedTypeName: jakarta.xml.soap.SOAPElementFactory
         newFullyQualifiedTypeName: jakarta.xml.soap.SOAPFactory
         ignoreDefinition: true
+---
+type: specs.openrewrite.org/v1beta/recipe
+name: org.openrewrite.java.migrate.jakarta.RemovedStateManagerMethods
+displayName: Use `StateManagementStrategy`
+description: >
+      Methods that were removed from the jakarta.faces.application.StateManager and javax.faces.application.StateManager classes in Jakarta Faces 4.0 are replaced
+      by jakarta.faces.view.StateManagementStrategy or javax.faces.view.StateManagementStrategy based on Jakarta10 migration in Faces 4.0
```

Comment:
Would you mind adding back ticks around code references here as well, similar to the display name? That helps the docs look best if class and package names are rendered as code.

Reply:
```suggestion
      by `jakarta.faces.view.StateManagementStrategy` or `javax.faces.view.StateManagementStrategy` based on Jakarta10 migration in Faces 4.0
```

Reply:
added in the next commit